src/test/java/com/epam/training/auction_backend/services/AuctionsServiceImplTestIT.java

```diff
@@ -0,0 +1,105 @@
+import java.util.List;
+
+public class AuctionsServiceImplTestIT extends CommonIntegrationTestSuit {
+
+    @Autowired
+    private AuctionRepository auctionRepository;
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private UsersService usersService;
+
+    private AuctionsServiceImpl auctionsService;
+
+    @Before
+    public void setUp() {
+        auctionsService = new AuctionsServiceImpl(auctionRepository);
+    }
+
+    @Test
+    @Transactional
+    public void getArchivedAuctions() {
+        final int expectedArchivedAuctionsSize = 2;
+
+        // create seller and winner for auctions
+        User seller = createUser("Seller", "Seller");
+        User winner = createUser("Winner", "Winner");
+
+        // create two auctions that must be saved in DB
+        auctionRepository.save(createAuction(false, seller, winner));
+        auctionRepository.save(createAuction(false, seller, winner));
+
+        // get archived auctions and check if size match
+        List<AuctionTransferObject> archivedAuctions = auctionsService.getArchivedAuctions();
+        final int actualArchivedAuctionsSize = archivedAuctions.size();
+        Assert.assertEquals("The getArchivedAuctions() method fetched wrong size of auctions",
+                expectedArchivedAuctionsSize, actualArchivedAuctionsSize);
+    }
+
+    @Test
+    @Transactional
+    public void getActiveAuctions() {
+        final int expectedActiveAuctionsSize = 2;
+
+        // create seller for auctions
+        User seller = createUser("Seller", "Seller");
+
+        // create two active auctions that must be saved in DB
+        auctionRepository.save(createAuction(true, seller));
+        auctionRepository.save(createAuction(true, seller));
+
+        // get archived auctions and check if size match
+        List<AuctionTransferObject> activeAuctions = auctionsService.getActiveAuctions();
+        final int actualActiveAuctionsSize = activeAuctions.size();
+        Assert.assertEquals("The getActiveAuctions() method fetched wrong size of auctions",
+                expectedActiveAuctionsSize, actualActiveAuctionsSize);
+    }
+
+    @Test
+    @Transactional
+    public void getAuctionById() {
+        // create seller for auctions
+        User seller = createUser("Seller", "Seller");
+
+        // create active auction that must be saved in DB and get ID of saved entity
+        final long expectedAuctionId = auctionRepository.save(createAuction(true, seller)).getId();
+
+        // get the ID of fetched object
+        final long actualAuctionId = auctionsService.getAuctionById(expectedAuctionId).get().getId();
+
+        Assert.assertEquals("The getAuctionById method fetched wrong object", expectedAuctionId, actualAuctionId);
+    }
+
+    private User createUser(String name, String password) {
+        return new User(name, password);
+    }
+
+    private Auction createAuction(boolean isActive, User seller, User winner) {
```

Comment:
Cleaner way would be to use createActiveAuction(User seller) and createArchivedAuction(User seller, User winner) methods.